index.d.ts

```diff
@@ -1,13 +1,6 @@
-import React from 'react'
-
-declare module 'react' {
-  interface StyleHTMLAttributes<T> extends HTMLAttributes<T> {
-    jsx?: boolean
-    global?: boolean
-  }
-}
-
 declare module 'styled-jsx' {
+  import React from 'react'
```

Comment:
Should we use `import('react').ReactNode` below instead?

Reply:
They both work, but `import React` will only work when I moved it into the styled-jsx scope. Seems it does have side effect?
I was just curious why you use `import('react')` cuz if there're more used types from react we need to write more `import()` so I tried this. I can change to `import('react')` since there's only 1 used type now.

Reply:
Yeah does seem there's a side-effect there, was mainly thinking to use it to limit potential conflicts